utils/web3_utils.ts

```diff
@@ -48,6 +48,43 @@ export class Web3Utils {
         return response.result;
     }
 
+    /**
+     * Returns the current blocktime in seconds.
+     *
+     * @returns {Promise<number>}
+     */
+    public async getCurrentBlockTime(): Promise<number> {
+        return this.web3.eth.getBlock("latest").timestamp;
```

Comment:
Maybe I'm missing something, but I'm not sure this would work as expected -- I believe `this.web3.eth.getBlock` doesn't return a promise (see [here](http://shawntabrizi.com/crypto/making-web3-js-work-asynchronously-javascript-promises-await/)) and, even if it did, I would think the promise would need to be resolved before the `timestamp` variable could be accessed.